src/components/Exercise/Edit.js

```diff
@@ -35,7 +35,7 @@ export default function Edit({
       <S.ButtonWrapper>
         <ButtonBackToHome />
         {(!disabled && <ButtonSave formState={formState} form={'edit'} />) || (
-          <Button form="edit" mla onClick={() => handleChange}>
+          <Button form="edit" primary mla onClick={() => handleChange}>
```

Comment:
No need to wrap `handleChange` in an arrow function

Reply:
fixed, ty 👍